src/pages/Accounts/Accounts.module.scss

```diff
@@ -0,0 +1,16 @@
+.header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    
+}
+
+.tab {
+    padding-top: 24px;
```

Comment:
remove this padding-top property, we want a accountContainer class that has gap 24px instead